www/js/control/general-settings.js

```diff
@@ -51,6 +52,8 @@ angular.module('emission.main.control',['emission.services',
       ionicDatePicker.openDatePicker(datepickerObject);
     };
 
+    $scope.carbonDatasetString = $translate.instant('carbon-dataset') + ": " + CarbonDatasetHelper.getCurrentCarbonDatasetName();
```

Comment:
Instead of using `CarbonDatasetHelper.getCurrentCarbonDatasetName()` would it be sensible to retrieve a translation for these too, based on locale/code (e.g. "US", "EU", DE")? 
This would imply that adding more datasets would require also adding more strings in the localization/translation file(s).

Reply:
I would assume so. I presume that `Kohle Datei: Germany` is going to look really weird. A workaround that would make it possible to add datasets without translations is to use the locale code directly (e.g. US/EU/DE/FR, etc). I think most people know country codes well enough to figure that out.

Thoughts? @PatGendre @jf87 @pgranato any thoughts from your side?

Reply:
Another option is 
```
if translate(countryCode) is defined
  use translation
else
  use countryCode
```

Reply:
I just tried requesting a translation string which doesn't exist:
```
var foo = $translate.instant('value-which-doesnt-exist');
``` 
instead of getting an `undefined` it apparently just broke (and the settings UI has all the Angular properties between curly braces). Is this intended and/or am I doing something wrong?

For now I will simply use country codes.

Reply:
@DO3B do you know what happens if we try to request a translation string that doesn't exist?

Reply:
It should just print the key because it didn't find any values. 

`$translate.instant('value-which-doesnt-exist'`) will return `value-which-doesnt-exist`